src/components/TimeseriesPreview/TimeseriesPreview.test.tsx

```diff
@@ -106,6 +106,42 @@ describe('TimeseriesChart', () => {
     expect(valueElement.text()).toEqual(`${value} psi`);
     expect(dateElement.text()).toEqual(expectDateString);
   });
+  it('should display provided custom name and description', async () => {
```

Comment:
This test suite fails on local and it should fail 'cause `nameFormatter` will return an empty string instead of provided. Mocked `singleTimeseries` contains `name` and `description` value for timeseries.